packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-actions.tsx

```diff
@@ -75,8 +82,13 @@ export const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
   onCollectionScanInsightActionButtonClick,
 }) => {
   const showInsights = usePreference('showInsights', React);
+  const enableAIExperience = usePreference('enableAIExperience', React);
+
   return (
     <div className={containerStyles}>
+      {enableAIExperience && showAIEntry && (
+        <AIExperienceEntry onClick={onShowAIInputClick} />
+      )}
```

Comment:
We are already rendering it inside the stages component, why not keep it all in one place and just push it to the right when there are stages in the pipeline?

Reply:
I found getting the styles to work nicely without doing it this way ended up being a lot of work. There might be a clean way to do it that's simple but I hadn't found it so I opted for this way.

Reply:
Feel free to keep it, sure, just FYI, it's two lines of css as far as I can see, maybe will come up helpful next time you need to style something similarly:

```diff
 const containerStyles = css({
+  flex: 1, // so that the pipeline takes all available space in the toolbar
   display: 'flex',
   gap: spacing[2],
   alignItems: 'center',
 });

+const buttonContainerStyles = css({
+  marginLeft: 'auto', // wrapper for your button so that it's pushed to the right in the layout
+});
```